data/pages.yaml

```diff
@@ -155,15 +155,15 @@ community:
   title: Join the Ansible community
   intro: Open-source and collaboration are at the heart of the Ansible community, helping more people experience the power of automation and work better and faster together.
   collaborate:
-    title: Collaborate
-    conduct: Review the Ansible Code of Conduct
-    help: Explore ways you can help
+    title: Collaborate with us
```

Comment:
@Andersson007 For me this wording is better because it directly appeals and humanizes things. Also "Collaborate" as a single word does not have the same "weight" as the other `title:` strings that contain multiple words. I feel like this change balances that out too.

Reply:
`with the community` maybe then? `with us` is also fine